### ISSUE TITLE
Different way of Colonization

### ISSUE THREAD

**Issue by PhelanBavaria** [PhelanBavaria/ancienttimeline#5]
The vanilla way of colonization will be used for the mod, but I had a second idea.
Most of Europe has nations placed on them, the issue is, people like Carthage made colonies which were basically just a harbor, they didn't cultivate the entire province. So both, the Carthaginians and the natives of that province have the right to own it.
My solution for that is a decision for colonizing nations to choose a province to colonize that way.
The goal province then gets a modifier "Carthaginian Colony" (or whatever the name) and the owner of that province has the right to accept them and gets a bonus to trade, or he refuses it.
After a while the owner of the colony gets a decision to buy the province from the owners, or alternatively, conquer it.
It would represent what happened far better than just leaving provinces empty or giving nations a CB to conquer them.

**Comment by Firesoul7** [comment]
It's a great idea but I don't think decisions are the way to go. How could
a person choose which province they want to establish a colony in? The
lists would be huge.

**Comment by qweytr** [comment]
I agree with Firesoul. There's nothing wrong with the vanilla colonization mechanic.

**Comment by PhelanBavaria** [comment]
As I said, the issue with the vanilla colonization mechanic is that you need to have a uncolonized province, which isn't always the case. Declaring war on the occupying nation to emulate colonization isn't very accurate.
Maybe rather a decision like "Search for a good spot to colonize", and you'll get an event with a chosen province, and you can say "Yes, I want to colonize there" or "No, keep looking".
If you still don't like this, then tell me how else you would handle this issue.

**Comment by Firesoul7** [comment]
Ideally there would be a way to build buildings on other people's
territory, then the owner of that territory would get the events you
described. This is likely impossible however

**Comment by qweytr** [comment]
I still think that the best solution is to have some uncolonized provinces in Hispania and Gaul. That would leave some room for colonization, but any wider expansion would require war.

**Comment by Firesoul7** [comment]
I like every tribe and nation currently ingame. I would not support any
removal of nations added by this mod. I also think that such a solution
would not accurately simulate the realities of the situation.

**Comment by PhelanBavaria** [comment]
Alright, I'll create a testing branch and try to write a series of decisions and events for it and you guys tell me how you like it.

**Comment by PhelanBavaria** [comment]
I didn't create a test branch yet because I still don't know how to do it.
It really is an issue to use the vanilla colonization system, since that is not how most colonies were established.
https://en.wikipedia.org/wiki/Colonies_in_antiquity
I think the kind of colonies that went away from their homeland to found a new city should be an event for the mother city. Then the new city gains a core in the new province, and if colonized, takes over the province. If it is colonized, then we could give them some event to take over the whole province at some point.
There are some reports of actual expeditions of nations (especially of Carthage), who found settlements under their rule. This might be represented by an event that gives that nation a colonist.
Perhaps we could give every nation such event/decision under certain circumstances.

**Comment by qweytr** [comment]
I agree that in most cases the colonizer shouldn't keep control of the colonized province. This doesn't mean the the vanilla colonization system is unsuitable. It just means that it needs to be tweaked a bit. 

For example most of the Greek states were city-states. They could have a specific city state government that makes owning other provinces harder and any far away colonies that they create would become independent after being founded.

We could also add some colonial nations to certain regions and reduce the max crown colonies to 0. Then even the very first colony would form its own colonial nation that could be given a high liberty desire.

Additional event driven colonization systems could be used as well, but I think that it should be possible for any country to get a colonist by picking some kind of colonization ideas.

**Comment by PhelanBavaria** [comment]
I guess we are on an agreement then. Let's use a combination of our ideas and see how it works and if something doesn't work well, we just delete it.
